presto-spi/src/test/java/io/prestosql/spi/block/BaseBlockEncodingTest.java

```diff
@@ -61,10 +62,38 @@ public void testNullAtTheBeginningAndEnd()
         roundTrip(null, null, randomValue(random), null, randomValue(random), null, null);
     }
 
+    @Test
+    public void testBlocksOf8()
+    {
+        Random random = getRandom();
+
+        Object[] values = Stream.of(
+                BlockFill.MIXED,
```

Comment:
static import these

Reply:
For some reason it is not possible. Static imports don't work within the same class.